src/payment/strategies/paypal-commerce/paypal-commerce-payment-initialize-options.ts

```diff
@@ -20,26 +20,30 @@ export type PaypalCommerceInitializeOptions = PaypalCommercePaymentInitializeOpt
  * service.initializePayment({
  *     methodId: 'paypalcommerce',
  *     paypalcommerce: {
- *         container: 'container',
+ *         container: '#container',
```

Comment:
just double-checking, does the `#` needs to be passed? most of our implementations require the `id` rather than the `selector`

Reply:
Changed id to selector.